delete.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
```

Comment:
:disappointed: so, thinking about it a bit as I worked on it, the name as an option could get a little confusing... I think we need to clean up how we are generating the help for the cogctl commands so that this can be clearer.